myStory.html

```diff
@@ -41,30 +42,34 @@ <h1>Happy Hearts Roaming</h1>
           <h1>My story</h1>
```

Comment:
The layout is broken after the css migration. 
Should this component be here?

Reply:
I haven't made any changes to the CSS for the 'My Story' page yet. The layout might look a bit different compared to the 'My Story' section on the landing page. I'll work on the CSS for the 'My Story' page once I've finished with the landing page.